jfr-daemon/src/main/java/com/newrelic/jfr/daemon/SetupUtils.java

```diff
@@ -66,6 +66,9 @@ public static DaemonConfig buildConfig() {
     builder.maybeEnv(EnvironmentVars.EVENTS_INGEST_URI, URI::create, builder::eventsUri);
     builder.maybeEnv(
         EnvironmentVars.JFR_SHARED_FILESYSTEM, Boolean::parseBoolean, builder::useSharedFilesystem);
+    builder.maybeEnv(
+        EnvironmentVars.JFR_USE_LICENSE_KEY, Boolean::parseBoolean, builder::useLicenseKey);
```

Comment:
It seems like `JFR_USE_LICENSE_KEY` does the same thing as the existing `USE_LICENSE_KEY`.